src/forms/NewStudyForm.js

```diff
@@ -1,351 +1,453 @@
+          </Container>
+        }
+        confirmButton={
+          <Button primary floated="right" type="submit" loading={isSubmitting}>
+            SUBMIT
+          </Button>
+        }
+      />
+    </>
+  );
+};
+
+const NewStudyForm = ({submitValue, apiErrors}) => {
+  const [confirmOpen, setConfirmOpen] = useState(false);
+  const [workflowType, setSelection] = useState([
+    'bwa_mem',
+    'gatk_haplotypecaller',
+  ]);
+  const [focused, setFocused] = useState('');
+  const [activeStep, setActiveStep] = useState('info');
+  return (
+    <Formik
+      initialValues={{
+        externalId: '',
+        name: '',
+        shortName: '',
+        description: '',
+        releaseDate: '',
+        version: '',
+        attribution: '',
+        anticipatedSamples: 0,
+        awardeeOrganization: '',
+      }}
+      validate={values => {
+        let errors = {};
+        if (!values.externalId) {
+          errors.externalId = 'Required';
+        }
+        if (!values.name) {
+          errors.name = 'Required';
+        }
+        if (values.anticipatedSamples < 0) {
+          errors.anticipatedSamples = 'Invalid number of anticipated samples';
+        }
+        if (values.version.length > 10) {
+          errors.version = 'Maximum 10 characters long';
+        }
+        return errors;
+      }}
+      onSubmit={(values, {setSubmitting}) => {
+        setSubmitting(false);
+        setConfirmOpen(false);
+        var inputObject = values;
+        if (values.releaseDate.length === 0) {
+          inputObject.releaseDate = null;
+        }
+        submitValue({input: inputObject, workflowType: workflowType});
+      }}
+    >
+      {formikProps => (
+        <Fragment>
+          {apiErrors && (
+            <Message
+              negative
+              icon="attention"
+              header="Error"
+              content={apiErrors}
+            />
+          )}
+          <Step.Group attached="top" fluid widths={3}>
+            <Step
+              active={activeStep === 'info'}
+              icon="info"
+              link
+              onClick={() => setActiveStep('info')}
+              title="Info"
+              description="General Study Details"
+            />
+            <Step
+              active={activeStep === 'external'}
+              icon="disk"
+              link
+              onClick={() => setActiveStep('external')}
+              title="External"
+              description="For dbGaP project"
+            />
+            <Step
+              active={activeStep === 'grant'}
+              icon="trophy"
+              link
+              onClick={() => setActiveStep('grant')}
+              title="Grant"
+              description="Kids First grant"
             />
-          </Form>
-        )}
-      </Formik>
-    </Segment>
+          </Step.Group>
+          <Segment padded clearing attached>
+            <Form onSubmit={formikProps.handleSubmit}>
+              {activeStep === 'info' && (
```

Comment:
if you structure the steps into an object as above you can compact this by doing 
```js
STUDY_STEPS[activeStep].comp({ ...props })
```

this way each step gets all the props but only uses the ones it needs